picamera2/picamera2.py

```diff
@@ -107,13 +110,19 @@ def _reset_flags(self) -> None:
         self._encoder = None
         self.request_callback = None
         self.completed_requests = []
-        self.lock = threading.Lock()  # protects the functions and completed_requests fields
+        self.lock = (
+            threading.Lock()
+        )  # protects the functions and completed_requests fields
         self.have_event_loop = False
```

Comment:
The original looks better and more concise in the original.

Reply:
I think it was just trying to maintain the comment on the same line here...